Doc/library/ast.rst

```diff
@@ -368,10 +368,19 @@ The following options are accepted:
    Specify what kind of code must be compiled, like the *mode* argument
    in :func:`parse`.
 
+.. cmdoption:: --no-type-comments
+
+   Don't parse type comments.
+
 .. cmdoption:: -a, --include-attributes
 
    Include attributes such as line numbers and column offsets.
 
+.. cmdoption:: -i <indent>
+               --indent <indent>
+
+   Indentation of nodes in AST.
```

Comment:
Ditto (versionadded).

The indentation is made of whitespaces, no? If yes, I suggest:

```suggestion
   Indentation of nodes in AST (number of spaces).
```